readme.md

```diff
@@ -326,7 +326,7 @@ if ! [ -d "$DRIVE_MOUNTPOINT" ]; then
     exit 1
 fi
 
-if ! grep -q " $DRIVE_MOUNTPOINT " /etc/fstab; then
+if ! grep -q "$DRIVE_MOUNTPOINT" /etc/fstab; then
```

Comment:
This is actually by intention or doesn't it work in your case?

Reply:
The script didn't work for me until I removed those spaces. It kept complaining that it could not find the drive in the fstab file even though I edited it in, so I took a look at the script and noticed that that line was the only one that had spaces within the quotes. Once I edited it it worked fine. I tested it on Debian 11. Does that answer your question?

Reply:
> Does that answer your question?

So I suppose you using tabs between the different entries?

Reply:
Sorry, I'm not sure what you mean.

Reply:
Glad to hear it :)